docs/docs/all-flags.md

```diff
@@ -111,21 +111,21 @@ An example job in YAML format:
 
 ```yaml
 spec:
-  engine: Docker
-  verifier: Noop
-  publisher: IPFS
-  docker:
-    image: ubuntu
-    entryPoint:
-      - echo
-    parameters:
-      - Hello
-      - World
-  outputs:
-    - name: outputs
-      path: /outputs
+    engine: Docker
+    verifier: Noop
+    publisher: IPFS
+    docker:
+        image: ubuntu
+        entryPoint:
+            - echo
+        parameters:
+            - Hello
+            - World
+    outputs:
+        - name: outputs
+          path: /outputs
 deal:
-  concurrency: 1
+    concurrency: 1
 ```
```

Comment:
yaml is space sensitive, and it looks like you have made modifications here (I assume they were unintentional) please ensure this yaml is still valid and revert this change if its not.